### DISCUSSION TITLE
about channels in hyper leadger fabric

### DISCUSSION THREAD

**Discussion by arunahaleeshwaran** [hyperledger/fabric#4450] [Q&A]
how to create private conversation in group of organization without using channels in hyper ledger version 3.0?

**Comment by jehanxaibahmed** [comment]
In Fabric v3.0 (as in earlier 2.x versions), the tool for keeping data private between a subset of orgs on the same channel — without provisioning a separate channel — is Private Data Collections (PDCs), not channels.

How it works: you define one or more collections in a `collections_config.json` and reference it when you install/approve/commit your chaincode definition. Each collection lists member orgs (`policy`), an `endorsementPolicy`, `requiredPeerCount`/`maxPeerCount` for private dissemination, and optionally `blockToLive` for automatic purging. Your chaincode calls `GetPrivateData`/`PutPrivateData` against that collection name. The private data itself is disseminated peer-to-peer only to member orgs' peers and stored in a separate private database on those peers; only a hash goes into the channel's public ledger, so non-member orgs on the same channel can verify a transaction happened but never see the payload.

So concretely: all orgs stay on one shared channel (one ordering service, one set of channel policies), but you scope sensitive data to a subset of orgs via the collection's membership list — that's the "private conversation within the group" you're describing, without the operational overhead of a second channel (separate genesis block, anchor peers, channel policies).

Docs: the Private data section of the Fabric documentation has a working `collections_config.json` walkthrough if you want to see the config end-to-end.

One caveat: PDCs still share the same channel-wide chaincode and access-control policies for everything else. If you need genuinely separate governance — a different orderer, different channel-level policies, a different set of participating orgs entirely — that's when a second channel is still the right tool. They solve different problems.
